### PR TITLE
Add the Wagtail tutorial to docs

### PR DESCRIPTION
**Pull request by zerolab**
This PR adds the “Your first Wagtail site” tutorial to the docs under “Getting started”.

Not entirely sure about the "Getting started" \ "Your first Wagtail site" pages, so suggestions are welcome.

### CODE REVIEW

File: docs/getting_started/tutorial.rst [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+        <div class="intro">{% self.intro %}</div>
+
+        {{ self.body | richtext }}
+
+    {% endblock %}
+
+.. figure:: ../_static/images/tutorial/tutorial_6.png
+   :alt: A blog post sample
+
+You can read more about using images in templates in the
```
Comment (kaedroho) [inline docs/getting_started/tutorial.rst]: ``/wagtail/docs/getting_started/tutorial.rst:213: WARNING: unknown document: ../topics/images``

File: docs/getting_started/tutorial.rst [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+
+   You can now access the administrative area at ``/admin``
+
+   .. figure:: ../_static/images/tutorial/tutorial_2.png
+      :alt: Administrative screen
+
+Extend the HomePage model
+-------------------------
+
+Out of the box you get a blank HomePage model and the core app has a
```
Comment (kaedroho) [inline docs/getting_started/tutorial.rst]: core => ``home``

File: docs/getting_started/tutorial.rst [hunk 3]
```diff
@@ -0,0 +1,384 @@
...
+    from wagtail.wagtailcore.fields import RichTextField
+    from wagtail.wagtailadmin.edit_handlers import FieldPanel
+
+
+    class HomePage(Page):
+        body = RichTextField(blank=True)
+
+        content_panels = Page.content_panels + [
+            FieldPanel('title', classname="full title"),
+            FieldPanel('intro')
```
Comment (kaedroho) [inline docs/getting_started/tutorial.rst]: intro => body

File: docs/getting_started/tutorial.rst [hunk 4]
```diff
@@ -0,0 +1,384 @@
...
+    from wagtail.wagtailcore.models import Page
+    from wagtail.wagtailcore.fields import RichTextField
+    from wagtail.wagtailadmin.edit_handlers import FieldPanel
+
+
+    class HomePage(Page):
+        body = RichTextField(blank=True)
+
+        content_panels = Page.content_panels + [
+            FieldPanel('title', classname="full title"),
```
Comment (kaedroho) [inline docs/getting_started/tutorial.rst]: This is already defined in Page.content_panels that's being extended

File: docs/getting_started/tutorial.rst [hunk 5]
```diff
@@ -0,0 +1,384 @@
...
+            FieldPanel('intro')
+        ]
+
+``body`` is defined as ``RichTextField``, a special Wagtail field. You
+can use any of the Django core fields. ``content_panels`` define the
```
Comment (kaedroho) [inline docs/getting_started/tutorial.rst]: Maybe make "Django core fields" a link to https://docs.djangoproject.com/en/1.8/ref/models/fields/?

File: docs/getting_started/tutorial.rst [hunk 6]
```diff
@@ -0,0 +1,384 @@
...
+suggestion by separating capital letters with underscores (e.g. HomePage
+becomes home\_page.html).
+
+.. code:: html+django
+
+    {% extends "base.html" %}
+
+    {% load static core_tags wagtailcore_tags %}
+
+    {% block body_class %}template-{{ self.get_verbose_name|slugify }}{% endblock %}
```
Comment (kaedroho) [inline docs/getting_started/tutorial.rst]: Could we use "template-homepage" as the classname? I know @davecranwell doesn't like this though, but I think it's much clearer.

File: docs/getting_started/tutorial.rst [hunk 7]
```diff
@@ -0,0 +1,384 @@
...
+        )
+
+        content_panels = Page.content_panels + [
+            FieldPanel('title', classname="full title"),
+            FieldPanel('date'),
+            FieldPanel('intro'),
+            FieldPanel('body', classname="full")
+        ]
+
+Add the new ``blog`` app to ``INSTALLED_APPS`` in
```
Comment (kaedroho) [inline docs/getting_started/tutorial.rst]: Maybe put this detail up in the creating the app part above. I think it makes sense to keep those two together

**Comment by zerolab** [comment]
@kaedroho thank you for the suggestions. Made the necessary tweaks.

File: docs/getting_started/tutorial.rst [hunk 8]
```diff
@@ -0,0 +1,376 @@
...
+
+Adjust your BlogPage template to output the image:
+
+.. code:: html+django
+
+    {% extends "base.html" %}
+
+    {% load static core_tags wagtailcore_tags wagtailimages_tags %}
+
+    {% block body_class %}template-{{ self.get_verbose_name|slugify }}{% endblock %}
```
Comment (kaedroho) [inline docs/getting_started/tutorial.rst]: ``template-blogpage`` ?

File: docs/getting_started/tutorial.rst [hunk 9]
```diff
@@ -0,0 +1,376 @@
...
+
+Extend ``blog_index_page.html`` to show related items
+
+.. code:: html+django
+
+    {% extends "base.html" %}
+
+    {% load static core_tags wagtailcore_tags %}
+
+    {% block body_class %}template-{{ self.get_verbose_name|slugify }}{% endblock %}
```
Comment (kaedroho) [inline docs/getting_started/tutorial.rst]: ``template-blogindexpage``

File: docs/getting_started/tutorial.rst [hunk 10]
```diff
@@ -0,0 +1,376 @@
...
+            PageChooserPanel('link_page'),
+        ]
+
+        class Meta:
+            abstract = True
+
+
+    # Related links
+
+    class RelatedLink(LinkFields):
```
Comment (kaedroho) [inline docs/getting_started/tutorial.rst]: Could we cut this example down a bit?

Comment (zerolab) [inline docs/getting_started/tutorial.rst]: I could cut out the `BlogPage` definition as that doesn't change. Not sure what else should be cut

File: docs/getting_started/tutorial.rst (hunk 1 above)
Comment (zerolab) [inline docs/getting_started/tutorial.rst]: sorted. Needed to be `../topics/images/index`

File: docs/getting_started/tutorial.rst (hunk 10 above)
Comment (zerolab) [inline docs/getting_started/tutorial.rst]: Removed the `BlogPage` definition for now.

**Comment by zerolab** [comment]
@kaedroho, I made the other suggested changes. Do let me know if there is anything else to change.

Thank you.

**Comment by kaedroho** [comment]
:+1:

**Pull request merged**